src/gui_egui/editor.rs

```diff
@@ -357,7 +364,10 @@ impl Editor {
                 _ => e.components.retain_mut(|c| {
                     let old_key = c.as_ref().get_id_ports().0;
                     let mut context = e.contexts.remove(&old_key).unwrap();
-                    let render_return = (*Rc::get_mut(c).unwrap()).render_editor(
+                    let debug_id = c.get_id_ports().0;
+                    let render_return = (*Rc::get_mut(c)
+                        .expect(&format!("More than one reference exist to {}, can't get mut, therefore not render editor", debug_id)))
```

Comment:
just #[allow] this lgtm

Reply:
fixed in 3c77dbf7ddc4bc5016e9541505ee179077ecf897